app/controllers/sessions_controller.rb

```diff
@@ -70,19 +70,19 @@ def logout
   protected
 
   def openid_authentication(openid_url, back_to)
-    #puts openid_url
-    authenticate_with_open_id(openid_url, :required => [:nickname, :email]) do |result, identity_url, registration|
+    # puts openid_url
+    authenticate_with_open_id(openid_url, required: %i(nickname email)) do |result, identity_url, registration|
       if result.successful?
         @user = User.find_by_identity_url(identity_url)
-        if not @user
+        unless @user
```

Comment:
If you wanted, I think you could use the **not-not** here and maybe other places (it is good practice as far as I know), like this:
 `!!@user`
The first ! is to turn _nill_ into _true_ and the second ! is to turn _true_ into _false_.